src/sentry/api/endpoints/organization_derive_code_mappings.py

```diff
@@ -45,15 +46,22 @@ def get(self, request: Request, organization: Organization) -> Response:
             return Response(status=status.HTTP_403_FORBIDDEN)
 
         stacktrace_filename = request.GET.get("stacktraceFilename")
-        installation, _ = get_installation(organization)
+        installation, _ = get_installation(organization)  # only returns GitHub integrations
         if not installation:
             return self.respond(
                 {"text": "Could not find this integration installed on your organization"},
                 status=status.HTTP_404_NOT_FOUND,
             )
 
         # This method is specific to the GithubIntegration
-        trees = installation.get_trees_for_org()  # type: ignore[attr-defined]
+        if not isinstance(installation, GitHubIntegration):
+            return self.respond(
+                {
+                    "text": f"The {installation.model.provider} integration does not support derived code mappings"
+                },
+                status=status.HTTP_400_BAD_REQUEST,
```

Comment:
Should we ever hit this case? If so, do we want to log here?

Reply:
no, the query for the installation only filters for github integrations. this fixes the typing but i need to return early and return something